python/py-flup/Portfile

```diff
@@ -20,22 +22,15 @@ long_description    Flup offer three sets of WSGI servers/gateways, which \
 homepage            https://www.saddi.com/software/flup
```

Comment:
```suggestion
homepage            https://www.saddi.com/software/flup/
```

Reply:
I know you didn't change this, but I cannot access this website without the trailing slash.

Reply:
okay, changed